course_discovery/apps/course_metadata/models.py

```diff
@@ -1414,6 +1414,17 @@ def first_enrollable_paid_seat_price(self):
 
         price = int(seats[0].price) if seats[0].price else None
         return price
+    
+    @property
+    def first_enrollable_paid_seat_price_currency(self):
+        # Sort in python to avoid an additional request to the database for order_by
```

Comment:
nit: you can replace # comment with docstrings.

Reply:
Replaced # with docstrings